server/app/routes/api/v1/va/serializers.js

```diff
@@ -97,7 +97,10 @@ function serializeAmbassador(ambassador) {
 
   // if claimees length exceeds threshold value, and stripe_1099_enabled is false, mark needs_w9_kyc as true
   const meets1099Requirements = ambassador.get('stripe_1099_enabled');
-  if (!meets1099Requirements) {
+  const hasPrimaryAccount = !!acct;
```

Comment:
The comment on line 98 is out of date; I'd just remove it as it's redundant with the code.